test/integration/imagestream_admission_test.go

```diff
@@ -86,7 +86,7 @@ func TestImageStreamTagsAdmission(t *testing.T) {
 		},
 	}
 	// we may hit cache with old limit, let's retry in such a case
-	err = retryOnQuotaExceeded(t, 1, func() error {
+	err = retryOnQuotaExceeded(t, 3, func() error {
```

Comment:
Implementation of this method should be updated to use `wait.ExponentialBackoff`.  If you think its cache latency, then we should be able to hit it again quickly.

Also, the old impl of passing `1` would never have retried, right?